### PR TITLE
Update forwarder-input-selectors.md

### PR DESCRIPTION
**Pull request by jcantrill**
Fix RE and modify struct to match impl

cc @alanconway

### CODE REVIEW

File: enhancements/cluster-logging/forwarder-input-selectors.md [hunk 1]
```diff
@@ -153,40 +153,36 @@ Following are the additions to the InputSpec:
...
     - name: my-infra
       infrastructure:
-        include: ["node","container"]
+        sources: ["node","container"]
 
 ```
 ```golang
    type Infrastructure struct {
-     Includes     []InfrastructureSource
+     Sources     []string
```
Comment (cahartma) [inline enhancements/cluster-logging/forwarder-input-selectors.md:161]: Need to change line:223 in the example as well.

File: enhancements/cluster-logging/forwarder-input-selectors.md [hunk 2]
```diff
@@ -195,7 +191,7 @@ The operator will validate resources upon reconciliation of a **ClusterLogForwar
 
 
 * The **ClusterLogForwarder** CR defines a valid spec
-* Input spec fields that are "globs" (i.e. Namespace, container) match RE: '`[a-zA-Z0-9\*]*`'
+* Input spec fields that are "globs" (i.e. Namespace, container) match RE: '`^[a-zA-Z0-9\*]*$`'
 * Input field 'selector' is a valid metav1.LabelSelector 
 * Input enum fields accept only the values listed
 * type "infrastructure" include specs at least one value
```
Comment (cahartma) [inline enhancements/cluster-logging/forwarder-input-selectors.md:197]: type "infrastructure" sources specifies at least one value

... same with line:198, do you want to change the word "include" to "sources"?

**Review by cahartma — COMMENTED** [review]
/lgtm

**Review by alanconway — APPROVED** [review]
:+1:

**Pull request merged**